docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1750 @@
+{
```

Comment:
<p><code>A cuspatial GeoDataFrame is a collection of data types</code>? I think it should be a collection of cudf Series and GeoSeries. cuspatial doesn't have built-in <code>geometry</code> dtype at this point so perhaps leave it out.</p><p> </p><p>Also a general suggestion on adding links. We should use it sporadically and be consistent. For example <code>cudf.Series</code> should just use one link.</p><p> </p><blockquote>One of the most important features of cuspatial is that it is highly integrated with <code>cudf</code>.&nbsp;You can use any <code>cudf</code> operation on cuspatial metadata columns, and most row 1:1 operations will work with a <code>geometry</code> column.</blockquote><p> </p><p>I'm not sure I understand. What is <code>cuspatial metadata columns</code> ? What is <code>row 1:1 operations</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
This comment corresponds to: `Geopandas and cudf integration` section.

Reply:
<p>Thanks. This looks much better now.</p>